applications/luci-app-usteer/htdocs/luci-static/resources/view/usteer/settings.js

```diff
@@ -0,0 +1,199 @@
+	o.optional    = true;
+	o.placeholder = 5;
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Value, 'seen_policy_timeout', _('seen policy timeout'), _('Maximum idle time of a station entry (ms) to be considered for policy decisions'));
+	o.optional    = true;
+	o.placeholder = 30000;
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Value, 'load_balancing_threshold', _('load_balancing_threshold'), _('Minimum number of stations delta between APs before load balancing policy is active'));
+	o.optional    = true;
+	o.placeholder = 5;
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Value, 'band_steering_threshold', _('band_steering_threshold'), _('Minimum number of stations delta between bands before band steering policy is active'));
+	o.optional    = true;
+	o.placeholder = 5;
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Value, 'band_steering_threshold', _('band_steering_threshold'), _('Minimum number of stations delta between bands before band steering policy is active'));
+	o.optional    = true;
+	o.placeholder = 5;
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Value, 'remote_update_interval', _('remote_update_interval'), _('Interval (ms) between sending state updates to other APs'));
+	o.optional    = true;
+	o.placeholder = 1000;
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Value, 'remote_node_timeout', _('remote_node_timeout'), _('Number of remote update intervals after which a remote-node is deleted'));
+	o.optional    = true;
+	o.placeholder = 10;
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Flag, 'assoc_steering', _('assoc_steering'), _('Allow rejecting assoc requests for steering purposes (0/1)'));
+	o.optional    = true;
+
+	o = s.option(form.Value, 'min_connect_snr', _('min_connect_snr'), _('Minimum signal-to-noise ratio or signal level (dBm) to allow connections'));
+	o.optional    = true;
+	o.placeholder = 0;
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Value, 'min_snr', _('min_snr'), _('Minimum signal-to-noise ratio or signal level (dBm) to remain connected'));
+	o.optional    = true;
+	o.placeholder = 0;
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Value, 'min_snr_kick_delay', _('min_snr_kick_delay'), _('Timeout after which a station with snr < min_snr will be kicked'));
+	o.optional    = true;
+	o.placeholder = 5000;
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Value, 'roam_process_timeout', _('roam_process_timeout'), _('Timeout (in ms) after which a association following a disassociation is not seen as a roam'));
+	o.optional    = true;
+	o.placeholder = 5000;
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Value, 'roam_scan_snr', _('roam_scan_snr'), _('Minimum signal-to-noise ratio or signal level (dBm) before attempting to trigger client scans for roam'));
+	o.optional    = true;
+	o.placeholder = 0;
+	o.datatype    = 'uinteger';
```

Comment:
If I'm not wrong, the dbm must be negative. At least I use like this in the `usteer` config file until now. For this value I use `-55` for example. With this config it only allows positive numbers.

Reply:
Ah good point. I will make it an integer